api/Console.json

```
@@ -289,6 +289,104 @@
           }
         }
       },
+      "debug": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/debug",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "substitution_strings": {
+          "__compat": {
+            "description": "Substitution strings",
+            "support": {
+              "chrome": {
+                "version_added": true,
+                "notes": "In version 28, if a negative value is passed to %d, it will be rounded down to the closest negative integer, so -0.1 becomes -1."
```

I thought this would read a little nicer, feel free to do something else with this:

```suggestion
                "notes": "In version 28, if a negative value is passed to <code>%d</code>, it will be rounded down to the closest negative integer. For example, <code>-0.1</code> becomes <code>-1</code>."
```

---

Hm. This isn't part of the work I did for this bug, but sure, I can do that. Feels like extending the scope of this PR though.

---

Sorry, I didn't realize you were copying from other data (I thought this was basically new)